docs/manual/scala/guide/broker/code/docs/mb/AnotherServiceImpl.scala

```diff
@@ -0,0 +1,25 @@
+package docs.mb
+
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import com.lightbend.lagom.scaladsl.api.ServiceCall
+
+//#inject-service
+class AnotherServiceImpl(helloService: HelloService) extends AnotherService {
+//#inject-service
+  
+  //#subscribe-to-topic
+  helloService.greetingsTopic()
+    .subscribe // <-- you get back a Subscriber instance
+    .atLeastOnce(
+    Flow.fromFunction(
+      doSomethingWithTheMessage
+    )
+  )
+  //#subscribe-to-topic
```

Comment:
Don't use `Flow.fromFunction`. I know the Java documentation uses it, but I'd rather do:

```scala
Flow[GreetingMessage].map { msg =>
  // Do something with the message
  Done
}
```

I think it makes it a little clearer to the user what's actually happening.